openmc/universe.py

```diff
@@ -1081,7 +1081,7 @@ def get_cell_instance(self, path, distribcell_index):
         # For 2D Lattices
         if len(self._dimension) == 2:
             offset = self._offsets[i[1]-1, i[2]-1, 0, distribcell_index-1]
-            offset += self._universes[i[1]][i[2]].get_cell_instance(path,
+            offset += self._universes[i[1]-1][i[2]-1].get_cell_instance(path,
```

Comment:
Correct - this code change is directly related to the bug fix I elaborated in my earlier comment.